docs/guides/writing-a-feature-service.md

```diff
@@ -150,7 +134,7 @@ const myFeatureServiceDefinition = {
   id: 'acme:my-feature-service',
 
   create(env) {
-    let count = env.config.initialCount || 0;
+    let count = 0;
```

Comment:
Shouldn't you continue to use a config here and show how to do it?

Reply:
No, I want to avoid the noise here. We have a separate doc for that.